http_server/queries/get_info/user.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+function user_select_by_name($pdo, $name) {
+	$stmt = $pdo->prepare('SELECT * FROM users WHERE name = ?');
+	$stmt->bindValue(1, $name, PDO::PARAM_STR);
+	$stmt->execute();
+	$row_count = $stmt->rowCount();
```

Comment:
It looks like `rowCount()` doesn't work with `SELECT` queries. 

http://php.net/manual/en/pdostatement.rowcount.php

Reply:
What if we did something like this?
```
$stmt = $pdo->prepare('SELECT *, COUNT(*) as row_count FROM users WHERE name = ?');
$stmt->bindValue(1, $name, PDO::PARAM_STR);
$stmt->execute();
$result = $stmt->fetchAll(PDO::FETCH_OBJ);
if ($result->row_count < 1) {
   // etc...
}
```

Reply:
rowCount() works fine on my localhost with a SELECT query, so I'm not sure why php.net says it doesn't work :thinking: